cluster/mesos/cluster.go

```diff
@@ -36,6 +37,7 @@ type Cluster struct {
 	TLSConfig           *tls.Config
 	options             *cluster.DriverOpts
 	offerTimeout        time.Duration
+	refuseTimeout       *float64
```

Comment:
@jimenez Would it be better to have its type as time.Duration, to be consistent with offerTimeout and taskCreationTimeout?